File: ruby/vristo-ruby-main/app/controllers/ui_controller.rb
class UiController < ApplicationController
    def components_tabs
        render layout: "default", :template => 'components/tabs'
    end

    def components_accordions
        render layout: "default", :template => 'components/accordions'
    end

    def components_modals
        render layout: "default", :template => 'components/modals'
    end

    def components_cards
        render layout: "default", :template => 'components/cards'
    end

    def components_carousel
        render layout: "default", :template => 'components/carousel'
    end

    def components_countdown
        render layout: "default", :template => 'components/countdown'
    end

    def components_counter
        render layout: "default", :template => 'components/counter'
    end

    def components_sweet_alert
        render layout: "default", :template => 'components/sweetalert'
    end

    def components_timeline
        render layout: "default", :template => 'components/timeline'
    end

    def components_notifications
        render layout: "default", :template => 'components/notifications'
    end

    def components_media_object
        render layout: "default", :template => 'components/media-object'
    end

    def components_list_group
        render layout: "default", :template => 'components/list-group'
    end

    def components_pricing
        render layout: "default", :template => 'components/pricing-table'
    end

    def components_lightbox
        render layout: "default", :template => 'components/lightbox'
    end

    def elements_alerts
        render layout: "default", :template => 'elements/alerts'
    end

    def elements_avatar
        render layout: "default", :template => 'elements/avatar'
    end

    def elements_badges
        render layout: "default", :template => 'elements/badges'
    end

    def elements_breadcrumbs
        render layout: "default", :template => 'elements/breadcrumbs'
    end

    def elements_buttons
        render layout: "default", :template => 'elements/buttons'
    end

    def elements_buttons_group
        render layout: "default", :template => 'elements/buttons-group'
    end

    def elements_color_library
        render layout: "default", :template => 'elements/color-library'
    end

    def elements_dropdown
        render layout: "default", :template => 'elements/dropdown'
    end

    def elements_infobox
        render layout: "default", :template => 'elements/infobox'
    end

    def elements_jumbotron
        render layout: "default", :template => 'elements/jumbotron'
    end

    def elements_loader
        render layout: "default", :template => 'elements/loader'
    end

    def elements_pagination
        render layout: "default", :template => 'elements/pagination'
    end

    def elements_popovers
        render layout: "default", :template => 'elements/popovers'
    end

    def elements_progress_bar
        render layout: "default", :template => 'elements/progress-bar'
    end

    def elements_search
        render layout: "default", :template => 'elements/search'
    end

    def elements_tooltips
        render layout: "default", :template => 'elements/tooltips'
    end

    def elements_treeview
        render layout: "default", :template => 'elements/treeview'
    end

    def elements_typography
        render layout: "default", :template => 'elements/typography'
    end

    def charts
        render layout: "default", :template => 'charts'
    end

    def widgets
        render layout: "default", :template => 'widgets'
    end

    def font_icons
        render layout: "default", :template => 'font-icons'
    end

    def drag_and_drop
        render layout: "default", :template => 'dragndrop'
    end
end

File: symfony/vristo-symfony-main/src/Controller/UIController.php
<?php

namespace App\Controller;

use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
use Symfony\Component\HttpFoundation\Response;
use Symfony\Component\Routing\Annotation\Route;

class UIController extends AbstractController
{
    public function ui_components_tabs(): Response
    {
        return $this->render('ui-components/tabs.html.twig');
    }

    public function ui_components_accordions(): Response
    {
        return $this->render('ui-components/accordions.html.twig');
    }

    public function ui_components_modals(): Response
    {
        return $this->render('ui-components/modals.html.twig');
    }

    public function ui_components_cards(): Response
    {
        return $this->render('ui-components/cards.html.twig');
    }

    public function ui_components_carousel(): Response
    {
        return $this->render('ui-components/carousel.html.twig');
    }

    public function ui_components_countdown(): Response
    {
        return $this->render('ui-components/countdown.html.twig');
    }

    public function ui_components_counter(): Response
    {
        return $this->render('ui-components/counter.html.twig');
    }

    public function ui_components_sweet_alert(): Response
    {
        return $this->render('ui-components/sweetalert.html.twig');
    }

    public function ui_components_timeline(): Response
    {
        return $this->render('ui-components/timeline.html.twig');
    }

    public function ui_components_notifications(): Response
    {
        return $this->render('ui-components/notifications.html.twig');
    }

    public function ui_components_media_object(): Response
    {
        return $this->render('ui-components/media-object.html.twig');
    }

    public function ui_components_list_group(): Response
    {
        return $this->render('ui-components/list-group.html.twig');
    }

    public function ui_components_pricing(): Response
    {
        return $this->render('ui-components/pricing-table.html.twig');
    }

    public function ui_components_lightbox(): Response
    {
        return $this->render('ui-components/lightbox.html.twig');
    }

    public function elements_alerts(): Response
    {
        return $this->render('elements/alerts.html.twig');
    }

    public function elements_avatar(): Response
    {
        return $this->render('elements/avatar.html.twig');
    }

    public function elements_badges(): Response
    {
        return $this->render('elements/badges.html.twig');
    }

    public function elements_breadcrumbs(): Response
    {
        return $this->render('elements/breadcrumbs.html.twig');
    }

    public function elements_buttons(): Response
    {
        return $this->render('elements/buttons.html.twig');
    }

    public function elements_button_group(): Response
    {
        return $this->render('elements/buttons-group.html.twig');
    }

    public function elements_color_library(): Response
    {
        return $this->render('elements/color-library.html.twig');
    }

    public function elements_dropdown(): Response
    {
        return $this->render('elements/dropdown.html.twig');
    }

    public function elements_infobox(): Response
    {
        return $this->render('elements/infobox.html.twig');
    }

    public function elements_jumbotron(): Response
    {
        return $this->render('elements/jumbotron.html.twig');
    }

    public function elements_loader(): Response
    {
        return $this->render('elements/loader.html.twig');
    }

    public function elements_pagination(): Response
    {
        return $this->render('elements/pagination.html.twig');
    }

    public function elements_popovers(): Response
    {
        return $this->render('elements/popovers.html.twig');
    }

    public function elements_progress_bar(): Response
    {
        return $this->render('elements/progress-bar.html.twig');
    }

    public function elements_search(): Response
    {
        return $this->render('elements/search.html.twig');
    }

    public function elements_tooltips(): Response
    {
        return $this->render('elements/tooltips.html.twig');
    }

    public function elements_treeview(): Response
    {
        return $this->render('elements/treeview.html.twig');
    }

    public function elements_typography(): Response
    {
        return $this->render('elements/typography.html.twig');
    }

    public function charts(): Response
    {
        return $this->render('charts.html.twig');
    }

    public function widgets(): Response
    {
        return $this->render('widgets.html.twig');
    }

    public function font_icons(): Response
    {
        return $this->render('font-icons.html.twig');
    }

    public function drag_and_drop(): Response
    {
        return $this->render('dragndrop.html.twig');
    }
}
